targets/ipq40xx-generic

```diff
@@ -73,6 +73,15 @@ device('gl.inet-gl-b1300', 'glinet_gl-b1300', {
 
 device('linksys-ea6350v3', 'linksys_ea6350v3')
 
+device('linksys-mr8300-dallas', 'linksys_mr8300', {
+	packages = { -- exclude until Gluon supports third radio
+		'-ath10k-firmware-qca9888-ct',
+		'-ath10k-board-qca9888',
+	},
+	factory = false,
+	broken = true, -- 3rd WiFi radio not supported (5GHz DFS channels 100+)
```

Comment:
Is the 3rd radio required for the upper channels, or can the other 5GHz radio be configured to all channels as well? If the latter is the case, this does not need to be set to `broken`.

Reply:
Oops sorry, I didn't read the previous comments in full. Broken is correct.